spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/README.adoc

```diff
@@ -31,6 +33,7 @@ Google App Engine.
 [source,yaml]
 ----
 spring.cloud.gcp.sql.database-name=[YOUR_CLOUD_SQL_DATABASE_NAME]
+# spring.cloud.gcp.sql.instance-connection-name OR spring.cloud.gcp.sql.jdbc-url (See below)
```

Comment:
Instead of this line, I'd prefer to explain this on the text description after this snippet. WDYT?